exporter/datadogexporter/factory.go

```diff
@@ -176,11 +175,8 @@ func (f *factory) createMetricsExporter(
 	c config.Exporter,
 ) (component.MetricsExporter, error) {
 
```

Comment:
Do we need these empty lines at the beginning of both `create*Exporter` funcs?

Reply:
Removed